cmd/cosign/cli/attest/attest_blob.go

```diff
@@ -126,6 +140,54 @@ func (c *AttestBlobCommand) Exec(ctx context.Context, artifactPath string) error
 		return errors.Wrap(err, "signing")
 	}
 
+	signedPayload := cosign.LocalSignedPayload{}
+	if c.TSAServerURL != "" {
+		clientTSA, err := tsaclient.GetTimestampClient(c.TSAServerURL)
+		if err != nil {
+			return fmt.Errorf("failed to create TSA client: %w", err)
+		}
+		rfc3161Timestamp, err := tsa.GetTimestampedSignature(sig, clientTSA)
+		if err != nil {
+			return err
+		}
+		if err := os.WriteFile(c.RFC3161TimestampPath, rfc3161Timestamp, 0600); err != nil {
```

Comment:
@haydentherapper I'll follow along on your PR for this, but to me it feels like if we do upload to a TSA we should always write the response out.

Reply:
Yea, I agree. You need to check that `RFC3161TimestampPath` is set. One case where we could relax this constraint is if we start uploading TSs to Rekor, then you don't need to write them, but that's something for the future. 

I'll make the same change to require both TSAServerUrl and RFC3161TimestampPath.

Reply:
sg!